.github/workflows/ci.yml

```
@@ -48,4 +49,69 @@ jobs:
+      - name: Install pluto
+        run: |
+          curl -L -o pluto.tar.gz \
+            https://github.com/FairwindsOps/pluto/releases/download/v5.20.3/pluto_5.20.3_linux_amd64.tar.gz
+          tar -xzf pluto.tar.gz pluto
+          sudo mv pluto /usr/local/bin/pluto
+      - name: Deprecated API check
+        run: |
+          pluto detect rendered.yaml --target-versions "k8s=v$K8S_VERSION"
+
+      - name: Upload rendered manifests
+          uses: actions/upload-artifact@v4
```

actionlint (1.7.12)

[error] could not parse as YAML: mapping values are not allowed in this context

(syntax-check)